### ISSUE TITLE
Output video not saved after running demo_video.py

### ISSUE THREAD

**Issue by MaramDjebbi** [face-analysis/emonet#23]
Hi,
I'm using the demo_video.py script with the following command:
`python3 demo_video.py --nclass 8 --video_path path/to/video.mp4 --output_path output.mp4`
The script runs successfully, prints predictions over frames, and doesn't raise any errors. However, no output video is saved.
Any help is appreciated!

Thanks in advanc

**Comment by Baoste** [comment]
Hi, I solve this problem.
At `demo_video.py`, line 284, replace:
replace
```python
out = cv2.VideoWriter(
    save_path,
    -1, 
    24.0, 
    (visualization_frames[0].shape[1], visualization_frames[0].shape[0]),
)
```
with
```python
fourcc = cv2.VideoWriter_fourcc(*'mp4v')
out = cv2.VideoWriter(
    save_path,
    fourcc,
    24.0,
    (visualization_frames[0].shape[1], visualization_frames[0].shape[0]),
)
```

**Comment by MaramDjebbi** [comment]
@Baoste thank you ! it works
